### PR TITLE
Added function that returns a random Float value between 0 an 1. Added a...

### PR DESCRIPTION
**Pull request by bettinaselig**
... function converting FloatBox2D into IntBox2D

### CODE REVIEW

File: source/sdk/math/Random.ooc [hunk 1]
```diff
@@ -25,7 +25,7 @@ Random: class {
 	state := static Time microtime()
 
 	/**
-	   :return: a pseudo-random number between INT_MIN and INT_MAX, generated using
+	   :return: a pseudo-random number between 0 and INT_MAX, generated using
```
Comment (davidhesselbom) [inline source/sdk/math/Random.ooc]: Was this comment wrong all along, or was there a change in behavior?

File: source/math/IntBox2D.ooc [hunk 2]
```diff
@@ -42,6 +42,7 @@ IntBox2D: cover {
 	bottomCenter ::= IntPoint2D new(this center x, this bottom)
 	empty ::= this size empty
 	init: func@ (=leftTop, =size)
+	init: func@ ~fromFloatBox2D (floatBox2D: FloatBox2D) {this init(floatBox2D left round() as Int, floatBox2D top round() as Int, floatBox2D width round() as Int, floatBox2D height round() as Int)}
```
Comment (davidhesselbom) [inline source/math/IntBox2D.ooc]: Leave a space after `{` and before `}`, please.

Comment (davidhesselbom) [inline source/math/IntBox2D.ooc]: Is the `round()` call necessary? What happens if you leave it out, will the resulting `Int` be floored, ceiled, or rounded?

File: source/sdk/math/Random.ooc (hunk 1 above)
Comment (bettinaselig) [inline source/sdk/math/Random.ooc]: The comment was wrong before. I checked it up and tested the function.

File: source/math/IntBox2D.ooc (hunk 2 above)
Comment (bettinaselig) [inline source/math/IntBox2D.ooc]: We did not see the function toIntBox2D(). That was the only reason why we added this constructor. 
Shall I take them away again? Or shall I add a corresponding constructor in the FloatBox2D class?

Comment (davidhesselbom) [inline source/math/IntBox2D.ooc]: I don't really see the point in having both functions, so I think you should remove this one and use the existing one instead, yes.

File: source/sdk/math/Random.ooc (hunk 1 above)
Comment (davidhesselbom) [inline source/sdk/math/Random.ooc]: Great, thanks!

**Comment by bettinaselig** [comment]
Done.

**Pull request merged**